cmake/morpheus_utils/package_config/matx/Configure_matx.cmake

```diff
@@ -22,8 +22,8 @@ function(morpheus_utils_configure_matx)
   list(APPEND CMAKE_MESSAGE_CONTEXT "matx")
 
   morpheus_utils_assert_cpm_initialized()
-  set(MATX_VERSION "0.4.0" CACHE STRING "Version of MatX to use")
-  set(MATX_TAG "v${MATX_VERSION}" CACHE STRING "Tag of MatX to use")
+  set(MATX_VERSION "0.3.0" CACHE STRING "Version of MatX to use")
+  set(MATX_TAG "015908f" CACHE STRING "Tag of MatX to use")
```

Comment:
Lets keep `0.4.0` but update the tag to the one that doesnt have a bug in it. Make and issue to get on 0.4.1 when its release.